docs/src/Tag.doc.js

```diff
@@ -10,7 +10,7 @@ const card = (c) => cards.push(c);
 card(
   <PageHeader
     name="Tag"
-    description="Tag is a object that holds text. It also has an x icon to remove it. Tags can appear within a [form field](/TextField#tagsExample) or as standalone components."
+    description="Tags are objects that hold text and have a cancel icon to remove them. They can appear within a [form field](/TextField#tagsExample) or as standalone components."
```

Comment:
maybe "a delete icon" instead?

Would also be great to link from here to all 3 options now that they're available: TextField, TextArea, and Tyepahead